serveradmin/backup.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import zipfile
+
+from datetime import datetime
+
+from django.conf import settings
+
+
+from ideasbox import __version__
+
+
+def make_name():
+    """Return backup formatted file name."""
+    return '{}.zip'.format('_'.join([
+        settings.IDEASBOX_ID,
+        __version__,
+        datetime.now().strftime(Backup.DATE_FORMAT)
+    ]))
+
+
+class Backup(object):
+
+    FORMAT = 'zip'
+    DATE_FORMAT = "%Y%m%d%H%M"
```

Comment:
Humm, my rationale was to go on the simpler (no space, comma, etc.) while human readable format.

Are you thinking about one specific?